cluster/util.sh

```diff
@@ -652,12 +652,12 @@ function kube::util::ensure-cfssl {
     kernel=$(uname -s)
     case "${kernel}" in
       Linux)
-        curl --retry 10 -L -o cfssl https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
-        curl --retry 10 -L -o cfssljson https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64
+        curl --retry 10 -L -o cfssl https://github.com/cloudflare/cfssl/releases/download/v1.5.0/cfssl_1.5.0_linux_amd64
```

Comment:
We usually perform a SHA-256 hash check

local cfssljson_1_5_0_amd64_hash = "ABCD..."

to make sure a download isn't randomly changing on us...

Reply:
Yes I would recommend adding it

Here is an example in OSS:

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#L26